mods/d2k/chrome.yaml

```diff
@@ -100,6 +100,7 @@ flags: buttons.png
 	smuggler: 111,114,23,21
 	fremen: 133,114,23,21
 	worm: 156,114,23,21
+	mercenary: 179,114,20,21
```

Comment:
Seems like we either have to keep it as is or change to artwork (again). Ping @x-a-n-a-x.